web/frontend/src/components/slam-2d-render.vue

```diff
@@ -295,20 +223,22 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
   const viewWidth = viewHeight * 2;
 
   const points = loader.parse(pointcloud.buffer);
+  pointsMaterial = points.material as THREE.PointsMaterial;
+  pointsMaterial.sizeAttenuation = false;
```

Comment:
`sizeAttenuation` determines if points become larger or smaller based on distance. It doesn't have any effect with an orthographic camera, but I still felt that for a 2D renderer it should be explicitly false.